specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -46,7 +46,39 @@ suppressions:
     reason: Suppress as instructed, this patch is for a abstract class and the type-discriminator needs to be required.
     where:
       - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/connections/{connectionName}"].patch.parameters[5].schema.properties.properties
-```
+  - code: ResourceNameRestriction
+    reason: Experience is the same as previous GA version, adding restriction will be a breaking change.
+    where:
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}"]
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/listNodes"]
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/listKeys"]
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/start"]
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/stop"]
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/restart"]
+  - code: AvoidAdditionalProperties
+    reason: Existing property in previous GA version.
+    where:
+      - $.definition.SparkJob.properties.conf
+      - $.definition.SparkJob.properties.environmentVariables
+      - $.definition.WorkspaceConnectionPropertiesV2.properties.metadata
+      - $.definition.CustomKeys.properties.keys
+  - code: LroLocationHeader
```

Comment:
I'm curious what exactly you're doing wrong with LroLocationHeader. Did you forget to declare a header that you actually return?

Reply:
we use to return 202 without a location header, I don't think we do that anymore, is removing this response code consider as a breaking change?

Reply:
So what do you do now, return it with a header? In that case you wouldn't need the suppression. Or you just return 200/204 success always?

Reply:
202 with header, let me update the swagger

Reply:
@TimLovellSmith Updated